### PR TITLE
Kafka1.0.0

### PR DESCRIPTION
**Pull request by marky-mark**
Kafka 1.0.0 upgrade

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: '3.3'
```
Comment (imduffy15) [inline docker-compose.yml]: How does this differ from the one in https://github.com/zalando-incubator/remora/blob/master/docker-compose/docker-compose.yml if we need updates to the docker-compose file please keep them all in the one file unless there's a different use case being tested.

Comment (marky-mark) [reply docker-compose.yml]: Forgot that existed, updated

**Comment by imduffy15** [comment]
👍

**Comment by marky-mark** [comment]
👍

**Pull request merged**